tensorflow/python/keras/activations.py

```diff
@@ -185,12 +200,21 @@ def softsign(x):
 @keras_export('keras.activations.swish')
 def swish(x):
   """Swish activation function.
-
+  Swish is an activation function which returns `x*sigmoid(x)`, swish tends to work
```

Comment:
Empty line above this one

Reply:
Also, let's rewrite this to look like two different statements.

Reply:
Added empty line

Reply:
I liked the comparison between this and the other activation function that you used to have. Can we add it back?